services/src/contexts/postgres.rs

```diff
@@ -3026,6 +3030,1186 @@ mod tests {
+                [
+                    crate::api::model::operators::OgrSourceDatasetTimeType::None,
+                    crate::api::model::operators::OgrSourceDatasetTimeType::Start {
+                        start_field: "start".to_string(),
+                        start_format: crate::api::model::operators::OgrSourceTimeFormat::Auto,
+                        duration: crate::api::model::operators::OgrSourceDurationSpec::Zero,
+                    },
+                    crate::api::model::operators::OgrSourceDatasetTimeType::StartEnd {
+                        start_field: "start".to_string(),
+                        start_format: crate::api::model::operators::OgrSourceTimeFormat::Auto,
+                        end_field: "end".to_string(),
+                        end_format: crate::api::model::operators::OgrSourceTimeFormat::Auto,
+                    },
+                    crate::api::model::operators::OgrSourceDatasetTimeType::StartDuration {
+                        start_field: "start".to_string(),
+                        start_format: crate::api::model::operators::OgrSourceTimeFormat::Auto,
+                        duration_field: "duration".to_string(),
+                    },
+                ],
+            )
+            .await;
+
+            test_type(
+                &pool,
+                "FormatSpecifics",
+                [crate::api::model::operators::FormatSpecifics::Csv {
+                    header: CsvHeader::Yes.into(),
+                }],
+            )
+            .await;
+
+            test_type(
+                &pool,
+                "OgrSourceColumnSpec",
+                [crate::api::model::operators::OgrSourceColumnSpec {
+                    format_specifics: Some(crate::api::model::operators::FormatSpecifics::Csv {
+                        header: CsvHeader::Auto.into(),
+                    }),
+                    x: "x".to_string(),
+                    y: Some("y".to_string()),
+                    int: vec!["int".to_string()],
+                    float: vec!["float".to_string()],
+                    text: vec!["text".to_string()],
+                    bool: vec!["bool".to_string()],
+                    datetime: vec!["datetime".to_string()],
+                    rename: Some(
+                        [
+                            ("xx".to_string(), "xx_renamed".to_string()),
+                            ("yx".to_string(), "yy_renamed".to_string()),
+                        ]
+                        .into(),
+                    ),
+                }],
+            )
+            .await;
+
+            test_type(
+                &pool,
+                "point[]",
+                [MultiPoint {
+                    coordinates: vec![
+                        Coordinate2D::new(0.0f64, 0.5).into(),
+                        Coordinate2D::new(2., 1.0).into(),
+                    ],
+                }],
+            )
+            .await;
+
+            // test_type(
```

Comment:
why is this commented out?

Reply:
[8d4d2e5](https://github.com/geo-engine/geoengine/pull/851/commits/8d4d2e52f63a17281ca50f8bd5f577d011338b98)

I removed the types as they were custom PATH and POINT impls.